source/containers/home-page/initState.json

```diff
@@ -0,0 +1,3 @@
+{
+  "userInfo": null
```

Comment:
Specify the data that the user will have as an object, and those values set them to null